libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -164,7 +164,15 @@ def draw_mermaid_png(
     Raises:
         ValueError: If an invalid draw method is provided.
     """
-    if draw_method == MermaidDrawMethod.PYPPETEER:
+    if draw_method == MermaidDrawMethod.PLAYWRIGHT:
```

Comment:
The draw_method should support inputs as strings and not require users to input enums